[sqlitecpp] Build failure on x64-linux

Package: sqlitecpp[core,sqlite]:x64-linux@3.3.2

**Host Environment**

- Host: x64-linux
- Compiler: GNU 15.1.1
-    vcpkg-tool version: 2025-04-16-f9b6c6917b23c1ccf16c1a9f015ebabf8f615045
    vcpkg-scripts version: 9dd12f8d79 2025-05-15 (12 hours ago)

**To Reproduce**

`vcpkg install sqlitecpp`

**Failure logs**

```
-- Using cached SRombauts-SQLiteCpp-3.3.2.tar.gz
-- Cleaning sources at /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/bobini/vcpkg/downloads/SRombauts-SQLiteCpp-3.3.2.tar.gz
-- Applying patch fix_dependency.patch
-- Applying patch add_runtime_destination.patch
-- Using source at /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean
-- Found external ninja('1.12.1').
-- Configuring x64-linux
CMake Warning at installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:344 (message):
  The following variables are not used in CMakeLists.txt:

      SQLITECPP_USE_STATIC_RUNTIME

  Please recheck them and remove the unnecessary options from the
  `vcpkg_cmake_configure` call.

  If these options should still be passed for whatever reason, please use the
  `MAYBE_UNUSED_VARIABLES` argument.
Call Stack (most recent call first):
  ports/sqlitecpp/portfile.cmake:21 (vcpkg_cmake_configure)
  scripts/ports.cmake:206 (include)


-- Building x64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /usr/bin/cmake --build . --config Debug --target install -- -v -j17
    Working Directory: /home/bobini/vcpkg/buildtrees/sqlitecpp/x64-linux-dbg
    See logs for more information:
      /home/bobini/vcpkg/buildtrees/sqlitecpp/install-x64-linux-dbg-out.log

Call Stack (most recent call first):
  installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/sqlitecpp/portfile.cmake:34 (vcpkg_cmake_install)
  scripts/ports.cmake:206 (include)



```

<details><summary>/home/bobini/vcpkg/buildtrees/sqlitecpp/install-x64-linux-dbg-out.log</summary>

```
Change Dir: '/home/bobini/vcpkg/buildtrees/sqlitecpp/x64-linux-dbg'

Run Build Command(s): /usr/bin/ninja -v -v -j17 install
[1/9] /usr/bin/c++ -DSQLITE_ENABLE_COLUMN_METADATA -I/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include -isystem /home/bobini/vcpkg/installed/x64-linux/include -fPIC -Wall -Wextra -Wpedantic -Wswitch-enum -Wshadow -Wno-long-long -g -std=gnu++11 -DSQLITECPP_HAS_MEM_STRUCT -fstack-protector -MD -MT CMakeFiles/SQLiteCpp.dir/src/Exception.cpp.o -MF CMakeFiles/SQLiteCpp.dir/src/Exception.cpp.o.d -o CMakeFiles/SQLiteCpp.dir/src/Exception.cpp.o -c /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Exception.cpp
[2/9] /usr/bin/c++ -DSQLITE_ENABLE_COLUMN_METADATA -I/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include -isystem /home/bobini/vcpkg/installed/x64-linux/include -fPIC -Wall -Wextra -Wpedantic -Wswitch-enum -Wshadow -Wno-long-long -g -std=gnu++11 -DSQLITECPP_HAS_MEM_STRUCT -fstack-protector -MD -MT CMakeFiles/SQLiteCpp.dir/src/Transaction.cpp.o -MF CMakeFiles/SQLiteCpp.dir/src/Transaction.cpp.o.d -o CMakeFiles/SQLiteCpp.dir/src/Transaction.cpp.o -c /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Transaction.cpp
FAILED: CMakeFiles/SQLiteCpp.dir/src/Transaction.cpp.o 
/usr/bin/c++ -DSQLITE_ENABLE_COLUMN_METADATA -I/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include -isystem /home/bobini/vcpkg/installed/x64-linux/include -fPIC -Wall -Wextra -Wpedantic -Wswitch-enum -Wshadow -Wno-long-long -g -std=gnu++11 -DSQLITECPP_HAS_MEM_STRUCT -fstack-protector -MD -MT CMakeFiles/SQLiteCpp.dir/src/Transaction.cpp.o -MF CMakeFiles/SQLiteCpp.dir/src/Transaction.cpp.o.d -o CMakeFiles/SQLiteCpp.dir/src/Transaction.cpp.o -c /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Transaction.cpp
In file included from /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:14,
                 from /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Database.h:14,
                 from /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Transaction.cpp:13:
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:132:39: error: ‘uint32_t’ does not name a type
  132 |     void bind(const int aIndex, const uint32_t      aValue);
      |                                       ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:20:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
   19 | #include <memory>
  +++ |+#include <cstdint>
   20 | 
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:132:10: error: ‘void SQLite::Statement::bind(int, int)’ cannot be overloaded with ‘void SQLite::Statement::bind(int, int32_t)’
  132 |     void bind(const int aIndex, const uint32_t      aValue);
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:128:10: note: previous declaration ‘void SQLite::Statement::bind(int, int32_t)’
  128 |     void bind(const int aIndex, const int32_t       aValue);
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:203:41: error: ‘uint32_t’ does not name a type
  203 |     void bind(const char* apName, const uint32_t        aValue)
      |                                         ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:203:41: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:203:10: error: ‘void SQLite::Statement::bind(const char*, int)’ cannot be overloaded with ‘void SQLite::Statement::bind(const char*, int32_t)’
  203 |     void bind(const char* apName, const uint32_t        aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:196:10: note: previous declaration ‘void SQLite::Statement::bind(const char*, int32_t)’
  196 |     void bind(const char* apName, const int32_t         aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:304:47: error: ‘uint32_t’ does not name a type
  304 |     void bind(const std::string& aName, const uint32_t        aValue)
      |                                               ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:304:47: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:304:10: error: ‘void SQLite::Statement::bind(const std::string&, int)’ cannot be overloaded with ‘void SQLite::Statement::bind(const std::string&, int32_t)’
  304 |     void bind(const std::string& aName, const uint32_t        aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:297:10: note: previous declaration ‘void SQLite::Statement::bind(const std::string&, int32_t)’
  297 |     void bind(const std::string& aName, const int32_t         aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:80:5: error: ‘uint32_t’ does not name a type
   80 |     uint32_t    getUInt() const noexcept;
      |     ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:15:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
   14 | #include <SQLiteCpp/Statement.h>
  +++ |+#include <cstdint>
   15 | #include <SQLiteCpp/Exception.h>
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:171:14: error: expected type-specifier before ‘uint8_t’
  171 |     operator uint8_t() const
      |              ^~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:179:14: error: expected type-specifier before ‘uint16_t’
  179 |     operator uint16_t() const
      |              ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:187:14: error: expected type-specifier before ‘uint32_t’
  187 |     operator uint32_t() const
      |              ^~~~~~~~
[3/9] /usr/bin/c++ -DSQLITE_ENABLE_COLUMN_METADATA -I/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include -isystem /home/bobini/vcpkg/installed/x64-linux/include -fPIC -Wall -Wextra -Wpedantic -Wswitch-enum -Wshadow -Wno-long-long -g -std=gnu++11 -DSQLITECPP_HAS_MEM_STRUCT -fstack-protector -MD -MT CMakeFiles/SQLiteCpp.dir/src/Savepoint.cpp.o -MF CMakeFiles/SQLiteCpp.dir/src/Savepoint.cpp.o.d -o CMakeFiles/SQLiteCpp.dir/src/Savepoint.cpp.o -c /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Savepoint.cpp
FAILED: CMakeFiles/SQLiteCpp.dir/src/Savepoint.cpp.o 
/usr/bin/c++ -DSQLITE_ENABLE_COLUMN_METADATA -I/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include -isystem /home/bobini/vcpkg/installed/x64-linux/include -fPIC -Wall -Wextra -Wpedantic -Wswitch-enum -Wshadow -Wno-long-long -g -std=gnu++11 -DSQLITECPP_HAS_MEM_STRUCT -fstack-protector -MD -MT CMakeFiles/SQLiteCpp.dir/src/Savepoint.cpp.o -MF CMakeFiles/SQLiteCpp.dir/src/Savepoint.cpp.o.d -o CMakeFiles/SQLiteCpp.dir/src/Savepoint.cpp.o -c /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Savepoint.cpp
In file included from /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:14,
                 from /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Database.h:14,
                 from /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Savepoint.cpp:15:
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:132:39: error: ‘uint32_t’ does not name a type
  132 |     void bind(const int aIndex, const uint32_t      aValue);
      |                                       ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:20:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
   19 | #include <memory>
  +++ |+#include <cstdint>
   20 | 
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:132:10: error: ‘void SQLite::Statement::bind(int, int)’ cannot be overloaded with ‘void SQLite::Statement::bind(int, int32_t)’
  132 |     void bind(const int aIndex, const uint32_t      aValue);
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:128:10: note: previous declaration ‘void SQLite::Statement::bind(int, int32_t)’
  128 |     void bind(const int aIndex, const int32_t       aValue);
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:203:41: error: ‘uint32_t’ does not name a type
  203 |     void bind(const char* apName, const uint32_t        aValue)
      |                                         ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:203:41: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:203:10: error: ‘void SQLite::Statement::bind(const char*, int)’ cannot be overloaded with ‘void SQLite::Statement::bind(const char*, int32_t)’
  203 |     void bind(const char* apName, const uint32_t        aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:196:10: note: previous declaration ‘void SQLite::Statement::bind(const char*, int32_t)’
  196 |     void bind(const char* apName, const int32_t         aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:304:47: error: ‘uint32_t’ does not name a type
  304 |     void bind(const std::string& aName, const uint32_t        aValue)
      |                                               ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:304:47: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:304:10: error: ‘void SQLite::Statement::bind(const std::string&, int)’ cannot be overloaded with ‘void SQLite::Statement::bind(const std::string&, int32_t)’
  304 |     void bind(const std::string& aName, const uint32_t        aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:297:10: note: previous declaration ‘void SQLite::Statement::bind(const std::string&, int32_t)’
  297 |     void bind(const std::string& aName, const int32_t         aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:80:5: error: ‘uint32_t’ does not name a type
   80 |     uint32_t    getUInt() const noexcept;
      |     ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:15:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
   14 | #include <SQLiteCpp/Statement.h>
  +++ |+#include <cstdint>
   15 | #include <SQLiteCpp/Exception.h>
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:171:14: error: expected type-specifier before ‘uint8_t’
  171 |     operator uint8_t() const
      |              ^~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:179:14: error: expected type-specifier before ‘uint16_t’
  179 |     operator uint16_t() const
      |              ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:187:14: error: expected type-specifier before ‘uint32_t’
  187 |     operator uint32_t() const
      |              ^~~~~~~~
[4/9] /usr/bin/c++ -DSQLITE_ENABLE_COLUMN_METADATA -I/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include -isystem /home/bobini/vcpkg/installed/x64-linux/include -fPIC -Wall -Wextra -Wpedantic -Wswitch-enum -Wshadow -Wno-long-long -g -std=gnu++11 -DSQLITECPP_HAS_MEM_STRUCT -fstack-protector -MD -MT CMakeFiles/SQLiteCpp.dir/src/Backup.cpp.o -MF CMakeFiles/SQLiteCpp.dir/src/Backup.cpp.o.d -o CMakeFiles/SQLiteCpp.dir/src/Backup.cpp.o -c /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Backup.cpp
FAILED: CMakeFiles/SQLiteCpp.dir/src/Backup.cpp.o 
/usr/bin/c++ -DSQLITE_ENABLE_COLUMN_METADATA -I/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include -isystem /home/bobini/vcpkg/installed/x64-linux/include -fPIC -Wall -Wextra -Wpedantic -Wswitch-enum -Wshadow -Wno-long-long -g -std=gnu++11 -DSQLITECPP_HAS_MEM_STRUCT -fstack-protector -MD -MT CMakeFiles/SQLiteCpp.dir/src/Backup.cpp.o -MF CMakeFiles/SQLiteCpp.dir/src/Backup.cpp.o.d -o CMakeFiles/SQLiteCpp.dir/src/Backup.cpp.o -c /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Backup.cpp
In file included from /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:14,
                 from /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Database.h:14,
                 from /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Backup.h:15,
                 from /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Backup.cpp:12:
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:132:39: error: ‘uint32_t’ does not name a type
  132 |     void bind(const int aIndex, const uint32_t      aValue);
      |                                       ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:20:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
   19 | #include <memory>
  +++ |+#include <cstdint>
   20 | 
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:132:10: error: ‘void SQLite::Statement::bind(int, int)’ cannot be overloaded with ‘void SQLite::Statement::bind(int, int32_t)’
  132 |     void bind(const int aIndex, const uint32_t      aValue);
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:128:10: note: previous declaration ‘void SQLite::Statement::bind(int, int32_t)’
  128 |     void bind(const int aIndex, const int32_t       aValue);
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:203:41: error: ‘uint32_t’ does not name a type
  203 |     void bind(const char* apName, const uint32_t        aValue)
      |                                         ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:203:41: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:203:10: error: ‘void SQLite::Statement::bind(const char*, int)’ cannot be overloaded with ‘void SQLite::Statement::bind(const char*, int32_t)’
  203 |     void bind(const char* apName, const uint32_t        aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:196:10: note: previous declaration ‘void SQLite::Statement::bind(const char*, int32_t)’
  196 |     void bind(const char* apName, const int32_t         aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:304:47: error: ‘uint32_t’ does not name a type
  304 |     void bind(const std::string& aName, const uint32_t        aValue)
      |                                               ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:304:47: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:304:10: error: ‘void SQLite::Statement::bind(const std::string&, int)’ cannot be overloaded with ‘void SQLite::Statement::bind(const std::string&, int32_t)’
  304 |     void bind(const std::string& aName, const uint32_t        aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:297:10: note: previous declaration ‘void SQLite::Statement::bind(const std::string&, int32_t)’
  297 |     void bind(const std::string& aName, const int32_t         aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:80:5: error: ‘uint32_t’ does not name a type
   80 |     uint32_t    getUInt() const noexcept;
      |     ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:15:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
   14 | #include <SQLiteCpp/Statement.h>
  +++ |+#include <cstdint>
   15 | #include <SQLiteCpp/Exception.h>
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:171:14: error: expected type-specifier before ‘uint8_t’
  171 |     operator uint8_t() const
      |              ^~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:179:14: error: expected type-specifier before ‘uint16_t’
  179 |     operator uint16_t() const
      |              ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:187:14: error: expected type-specifier before ‘uint32_t’
  187 |     operator uint32_t() const
      |              ^~~~~~~~
[5/9] /usr/bin/c++ -DSQLITE_ENABLE_COLUMN_METADATA -I/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include -isystem /home/bobini/vcpkg/installed/x64-linux/include -fPIC -Wall -Wextra -Wpedantic -Wswitch-enum -Wshadow -Wno-long-long -g -std=gnu++11 -DSQLITECPP_HAS_MEM_STRUCT -fstack-protector -MD -MT CMakeFiles/SQLiteCpp.dir/src/Statement.cpp.o -MF CMakeFiles/SQLiteCpp.dir/src/Statement.cpp.o.d -o CMakeFiles/SQLiteCpp.dir/src/Statement.cpp.o -c /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Statement.cpp
FAILED: CMakeFiles/SQLiteCpp.dir/src/Statement.cpp.o 
/usr/bin/c++ -DSQLITE_ENABLE_COLUMN_METADATA -I/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include -isystem /home/bobini/vcpkg/installed/x64-linux/include -fPIC -Wall -Wextra -Wpedantic -Wswitch-enum -Wshadow -Wno-long-long -g -std=gnu++11 -DSQLITECPP_HAS_MEM_STRUCT -fstack-protector -MD -MT CMakeFiles/SQLiteCpp.dir/src/Statement.cpp.o -MF CMakeFiles/SQLiteCpp.dir/src/Statement.cpp.o.d -o CMakeFiles/SQLiteCpp.dir/src/Statement.cpp.o -c /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Statement.cpp
In file included from /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Statement.cpp:11:
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:132:39: error: ‘uint32_t’ does not name a type
  132 |     void bind(const int aIndex, const uint32_t      aValue);
      |                                       ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:20:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
   19 | #include <memory>
  +++ |+#include <cstdint>
   20 | 
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:132:10: error: ‘void SQLite::Statement::bind(int, int)’ cannot be overloaded with ‘void SQLite::Statement::bind(int, int32_t)’
  132 |     void bind(const int aIndex, const uint32_t      aValue);
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:128:10: note: previous declaration ‘void SQLite::Statement::bind(int, int32_t)’
  128 |     void bind(const int aIndex, const int32_t       aValue);
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:203:41: error: ‘uint32_t’ does not name a type
  203 |     void bind(const char* apName, const uint32_t        aValue)
      |                                         ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:203:41: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:203:10: error: ‘void SQLite::Statement::bind(const char*, int)’ cannot be overloaded with ‘void SQLite::Statement::bind(const char*, int32_t)’
  203 |     void bind(const char* apName, const uint32_t        aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:196:10: note: previous declaration ‘void SQLite::Statement::bind(const char*, int32_t)’
  196 |     void bind(const char* apName, const int32_t         aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:304:47: error: ‘uint32_t’ does not name a type
  304 |     void bind(const std::string& aName, const uint32_t        aValue)
      |                                               ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:304:47: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:304:10: error: ‘void SQLite::Statement::bind(const std::string&, int)’ cannot be overloaded with ‘void SQLite::Statement::bind(const std::string&, int32_t)’
  304 |     void bind(const std::string& aName, const uint32_t        aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:297:10: note: previous declaration ‘void SQLite::Statement::bind(const std::string&, int32_t)’
  297 |     void bind(const std::string& aName, const int32_t         aValue)
      |          ^~~~
In file included from /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Database.h:14,
                 from /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Statement.cpp:13:
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:80:5: error: ‘uint32_t’ does not name a type
   80 |     uint32_t    getUInt() const noexcept;
      |     ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:1:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
  +++ |+#include <cstdint>
    1 | /**
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:171:14: error: expected type-specifier before ‘uint8_t’
  171 |     operator uint8_t() const
      |              ^~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:179:14: error: expected type-specifier before ‘uint16_t’
  179 |     operator uint16_t() const
      |              ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:187:14: error: expected type-specifier before ‘uint32_t’
  187 |     operator uint32_t() const
      |              ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Statement.cpp:91:46: error: ‘uint32_t’ does not name a type
   91 | void Statement::bind(const int aIndex, const uint32_t aValue)
      |                                              ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Statement.cpp:19:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
   18 | #include <sqlite3.h>
  +++ |+#include <cstdint>
   19 | 
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Statement.cpp:91:6: error: redefinition of ‘void SQLite::Statement::bind(int, int)’
   91 | void Statement::bind(const int aIndex, const uint32_t aValue)
      |      ^~~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Statement.cpp:84:6: note: ‘void SQLite::Statement::bind(int, int32_t)’ previously defined here
   84 | void Statement::bind(const int aIndex, const int32_t aValue)
      |      ^~~~~~~~~
[6/9] /usr/bin/c++ -DSQLITE_ENABLE_COLUMN_METADATA -I/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include -isystem /home/bobini/vcpkg/installed/x64-linux/include -fPIC -Wall -Wextra -Wpedantic -Wswitch-enum -Wshadow -Wno-long-long -g -std=gnu++11 -DSQLITECPP_HAS_MEM_STRUCT -fstack-protector -MD -MT CMakeFiles/SQLiteCpp.dir/src/Column.cpp.o -MF CMakeFiles/SQLiteCpp.dir/src/Column.cpp.o.d -o CMakeFiles/SQLiteCpp.dir/src/Column.cpp.o -c /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Column.cpp
FAILED: CMakeFiles/SQLiteCpp.dir/src/Column.cpp.o 
/usr/bin/c++ -DSQLITE_ENABLE_COLUMN_METADATA -I/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include -isystem /home/bobini/vcpkg/installed/x64-linux/include -fPIC -Wall -Wextra -Wpedantic -Wswitch-enum -Wshadow -Wno-long-long -g -std=gnu++11 -DSQLITECPP_HAS_MEM_STRUCT -fstack-protector -MD -MT CMakeFiles/SQLiteCpp.dir/src/Column.cpp.o -MF CMakeFiles/SQLiteCpp.dir/src/Column.cpp.o.d -o CMakeFiles/SQLiteCpp.dir/src/Column.cpp.o -c /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Column.cpp
In file included from /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:14,
                 from /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Column.cpp:11:
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:132:39: error: ‘uint32_t’ does not name a type
  132 |     void bind(const int aIndex, const uint32_t      aValue);
      |                                       ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:20:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
   19 | #include <memory>
  +++ |+#include <cstdint>
   20 | 
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:132:10: error: ‘void SQLite::Statement::bind(int, int)’ cannot be overloaded with ‘void SQLite::Statement::bind(int, int32_t)’
  132 |     void bind(const int aIndex, const uint32_t      aValue);
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:128:10: note: previous declaration ‘void SQLite::Statement::bind(int, int32_t)’
  128 |     void bind(const int aIndex, const int32_t       aValue);
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:203:41: error: ‘uint32_t’ does not name a type
  203 |     void bind(const char* apName, const uint32_t        aValue)
      |                                         ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:203:41: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:203:10: error: ‘void SQLite::Statement::bind(const char*, int)’ cannot be overloaded with ‘void SQLite::Statement::bind(const char*, int32_t)’
  203 |     void bind(const char* apName, const uint32_t        aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:196:10: note: previous declaration ‘void SQLite::Statement::bind(const char*, int32_t)’
  196 |     void bind(const char* apName, const int32_t         aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:304:47: error: ‘uint32_t’ does not name a type
  304 |     void bind(const std::string& aName, const uint32_t        aValue)
      |                                               ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:304:47: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:304:10: error: ‘void SQLite::Statement::bind(const std::string&, int)’ cannot be overloaded with ‘void SQLite::Statement::bind(const std::string&, int32_t)’
  304 |     void bind(const std::string& aName, const uint32_t        aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:297:10: note: previous declaration ‘void SQLite::Statement::bind(const std::string&, int32_t)’
  297 |     void bind(const std::string& aName, const int32_t         aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:80:5: error: ‘uint32_t’ does not name a type
   80 |     uint32_t    getUInt() const noexcept;
      |     ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:15:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
   14 | #include <SQLiteCpp/Statement.h>
  +++ |+#include <cstdint>
   15 | #include <SQLiteCpp/Exception.h>
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:171:14: error: expected type-specifier before ‘uint8_t’
  171 |     operator uint8_t() const
      |              ^~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:179:14: error: expected type-specifier before ‘uint16_t’
  179 |     operator uint16_t() const
      |              ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:187:14: error: expected type-specifier before ‘uint32_t’
  187 |     operator uint32_t() const
      |              ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Column.cpp:59:1: error: ‘uint32_t’ does not name a type
   59 | uint32_t Column::getUInt() const noexcept
      | ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Column.cpp:16:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
   15 | #include <iostream>
  +++ |+#include <cstdint>
   16 | 
[7/9] /usr/bin/c++ -DSQLITE_ENABLE_COLUMN_METADATA -I/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include -isystem /home/bobini/vcpkg/installed/x64-linux/include -fPIC -Wall -Wextra -Wpedantic -Wswitch-enum -Wshadow -Wno-long-long -g -std=gnu++11 -DSQLITECPP_HAS_MEM_STRUCT -fstack-protector -MD -MT CMakeFiles/SQLiteCpp.dir/src/Database.cpp.o -MF CMakeFiles/SQLiteCpp.dir/src/Database.cpp.o.d -o CMakeFiles/SQLiteCpp.dir/src/Database.cpp.o -c /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Database.cpp
FAILED: CMakeFiles/SQLiteCpp.dir/src/Database.cpp.o 
/usr/bin/c++ -DSQLITE_ENABLE_COLUMN_METADATA -I/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include -isystem /home/bobini/vcpkg/installed/x64-linux/include -fPIC -Wall -Wextra -Wpedantic -Wswitch-enum -Wshadow -Wno-long-long -g -std=gnu++11 -DSQLITECPP_HAS_MEM_STRUCT -fstack-protector -MD -MT CMakeFiles/SQLiteCpp.dir/src/Database.cpp.o -MF CMakeFiles/SQLiteCpp.dir/src/Database.cpp.o.d -o CMakeFiles/SQLiteCpp.dir/src/Database.cpp.o -c /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Database.cpp
In file included from /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:14,
                 from /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Database.h:14,
                 from /home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/src/Database.cpp:11:
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:132:39: error: ‘uint32_t’ does not name a type
  132 |     void bind(const int aIndex, const uint32_t      aValue);
      |                                       ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:20:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
   19 | #include <memory>
  +++ |+#include <cstdint>
   20 | 
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:132:10: error: ‘void SQLite::Statement::bind(int, int)’ cannot be overloaded with ‘void SQLite::Statement::bind(int, int32_t)’
  132 |     void bind(const int aIndex, const uint32_t      aValue);
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:128:10: note: previous declaration ‘void SQLite::Statement::bind(int, int32_t)’
  128 |     void bind(const int aIndex, const int32_t       aValue);
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:203:41: error: ‘uint32_t’ does not name a type
  203 |     void bind(const char* apName, const uint32_t        aValue)
      |                                         ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:203:41: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:203:10: error: ‘void SQLite::Statement::bind(const char*, int)’ cannot be overloaded with ‘void SQLite::Statement::bind(const char*, int32_t)’
  203 |     void bind(const char* apName, const uint32_t        aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:196:10: note: previous declaration ‘void SQLite::Statement::bind(const char*, int32_t)’
  196 |     void bind(const char* apName, const int32_t         aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:304:47: error: ‘uint32_t’ does not name a type
  304 |     void bind(const std::string& aName, const uint32_t        aValue)
      |                                               ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:304:47: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:304:10: error: ‘void SQLite::Statement::bind(const std::string&, int)’ cannot be overloaded with ‘void SQLite::Statement::bind(const std::string&, int32_t)’
  304 |     void bind(const std::string& aName, const uint32_t        aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Statement.h:297:10: note: previous declaration ‘void SQLite::Statement::bind(const std::string&, int32_t)’
  297 |     void bind(const std::string& aName, const int32_t         aValue)
      |          ^~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:80:5: error: ‘uint32_t’ does not name a type
   80 |     uint32_t    getUInt() const noexcept;
      |     ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:15:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
   14 | #include <SQLiteCpp/Statement.h>
  +++ |+#include <cstdint>
   15 | #include <SQLiteCpp/Exception.h>
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:171:14: error: expected type-specifier before ‘uint8_t’
  171 |     operator uint8_t() const
      |              ^~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:179:14: error: expected type-specifier before ‘uint16_t’
  179 |     operator uint16_t() const
      |              ^~~~~~~~
/home/bobini/vcpkg/buildtrees/sqlitecpp/src/3.3.2-2fa7969843.clean/include/SQLiteCpp/Column.h:187:14: error: expected type-specifier before ‘uint32_t’
  187 |     operator uint32_t() const
      |              ^~~~~~~~
ninja: build stopped: subcommand failed.
```
</details>

--- a/ports/sqlitecpp/fix_dependency.patch
+++ b/ports/sqlitecpp/fix_dependency.patch
@@ -1,21 +1,21 @@
 diff --git a/CMakeLists.txt b/CMakeLists.txt
-index df5693d..e5723d0 100644
+index 50362fb..5d7c22f 100644
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -295,9 +295,9 @@ else (SQLITECPP_INTERNAL_SQLITE)
+@@ -325,9 +325,9 @@ else (SQLITECPP_INTERNAL_SQLITE)
              target_link_libraries(SQLiteCpp PRIVATE ${sqlcipher_LIBRARY})
          endif()
      else()
 -        find_package (SQLite3 REQUIRED)
 +        find_package (unofficial-sqlite3 CONFIG)
-         message(STATUS "Link to sqlite3 system library")
+         message(STATUS "Link to sqlite3 system library ${SQLite3_VERSION}")
 -        target_link_libraries(SQLiteCpp PUBLIC SQLite::SQLite3)
 +        target_link_libraries(SQLiteCpp PRIVATE unofficial::sqlite3::sqlite3)
          if(SQLite3_VERSION VERSION_LESS "3.19")
              set_target_properties(SQLiteCpp PROPERTIES COMPILE_FLAGS "-DSQLITECPP_HAS_MEM_STRUCT")
          endif()
 diff --git a/cmake/SQLiteCppConfig.cmake.in b/cmake/SQLiteCppConfig.cmake.in
-index 2b48df4..d0feda9 100644
+index 7d0941c..d0feda9 100644
 --- a/cmake/SQLiteCppConfig.cmake.in
 +++ b/cmake/SQLiteCppConfig.cmake.in
 @@ -1,6 +1,6 @@

--- a/ports/sqlitecpp/portfile.cmake
+++ b/ports/sqlitecpp/portfile.cmake
@@ -2,7 +2,7 @@ vcpkg_from_github(OUT_SOURCE_PATH SOURCE_PATH
     REPO "SRombauts/SQLiteCpp"
     REF ${VERSION}
     HEAD_REF master
-    SHA512 e65695765756570d3b2df9151c968f76d434b9ec1dbcffa71e6b58a0cad005dc60c03ac826269d0292ecda2ad2abfbf2acd1cb71fb11967f53ff58140ea133fd
+    SHA512 23193108faaca4c1c7b0a05178bfdbb772a4e14dc145af1b3a7b35efe05a94b07200bdd5551afde44ab5eb3c6aeabbd70cfb0d710dc2750a8280e06fba94c494
     PATCHES
         fix_dependency.patch
         add_runtime_destination.patch

--- a/ports/sqlitecpp/vcpkg.json
+++ b/ports/sqlitecpp/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "sqlitecpp",
-  "version": "3.3.2",
+  "version": "3.3.3",
   "description": "SQLiteC++ (SQLiteCpp) is a smart and easy to use C++ SQLite3 wrapper.",
   "homepage": "https://github.com/SRombauts/SQLiteCpp",
   "license": "MIT",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -8957,7 +8957,7 @@
       "port-version": 0
     },
     "sqlitecpp": {
-      "baseline": "3.3.2",
+      "baseline": "3.3.3",
       "port-version": 0
     },
     "sqlpp11": {

--- a/versions/s-/sqlitecpp.json
+++ b/versions/s-/sqlitecpp.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "7d0741a4558bd9187854a03140417fcdf2679941",
+      "version": "3.3.3",
+      "port-version": 0
+    },
+    {
       "git-tree": "5ae20f010448408508d0ec198f3166f6f9794f69",
       "version": "3.3.2",
       "port-version": 0

